CONTRIBUTING.md

```diff
@@ -1,37 +1,99 @@
+
+We have an official message board with a detailed FAQ and where the community chimes in with helpful advice if you have questions.
+
+* [The official Netlify community forum](https://answers.netlify.com/)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out [the required template](https://github.com/netlify/netlify-plugin-nextjs/blob/main/.github/ISSUE_TEMPLATE/bug_report.md), the information it asks for helps us resolve issues faster.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### Before Submitting A Bug Report
+
+* **Check the [forum](https://answers.netlify.com/)** for a list of common questions and problems related to this plugin.
+* **Perform a [cursory search](https://github.com/netlify/netlify-plugin-nextjs/search?q=)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Bug Report?
 
-If your pull request is related to a typo or the documentation being unclear, please click on the relevant page's `Edit`
-button (pencil icon) and directly suggest a correction instead.
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue and provide the following information by filling in [the template](https://github.com/netlify/netlify-plugin-nextjs/blob/main/.github/ISSUE_TEMPLATE/bug_report.md).
 
-This project was made with ❤️. The simplest way to give back is by starring and sharing it online.
+Explain the problem and include additional details to help maintainers reproduce the problem:
 
-Everyone is welcome regardless of personal background. We enforce a [Code of conduct](CODE_OF_CONDUCT.md) in order to
-promote a positive and inclusive environment.
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible. When listing steps, **don't just say what you did, but explain how you did it**.
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
```

Comment:
not that many people will read this in depth vs the github issue template, but we prob do want to emphasize that simple repos that reproduce the issue are preferred, and simple means anyone can clone and deploy (aka no api keys, no db setup, etc)